packages/lti-client/src/libs/post_message/platform_storage.ts

```diff
@@ -0,0 +1,89 @@
+import { PostMessageClient } from "./client";
```

Comment:
This works with all platforms that support LTI not just Canvas. I wonder if it should go in lti-client?

Reply:
It is in `lti-client` :D